generating_an_expression_matrix/README.md

```diff
@@ -61,98 +61,120 @@ This is what the your history will look like:
+![droplet barcode plot tool](droplet_barcode_tool-2.png)
 
-You'll end up with a plot like: 
+You'll end up with a plot like:
 
 ![barcode plot from raw barcode counts](barcodes_raw.png)
 
-This is our own formulation of the barcode plot based on a [discussion](https://github.com/COMBINE-lab/salmon/issues/362#issuecomment-490160480) we had with community members. The left hand plot is the main one, showing the counts for individual cell barcodes ranked from high to low. We expect a sharp drop-off between cell-containing droplets and ones that are empty or contain only cell debris. The right hand plot is a density from the first one, and the thresholds are generated either using [dropletUtils](https://bioconductor.org/packages/release/bioc/html/DropletUtils.html) or by the method described in that discussion. We could use any of these thresholds to select cells, assuming that anything with fewer counts is not a valid cell. By default, Alevin does something similar, and we can learn something about that by plotting just the barcodes Alevin retains. Go back and re-run the droplet barcode plot, this time selecting MTX input (quants_mat.mtx.gz). You will need to select the option to assume cells are by row (more on that later). 
+This is our own formulation of the barcode plot based on a [discussion](https://github.com/COMBINE-lab/salmon/issues/362#issuecomment-490160480) we had with community members. The left hand plot is the main one, showing the UMI counts for individual cell barcodes ranked from high to low. We expect a sharp drop-off between cell-containing droplets and ones that are empty or contain only cell debris. The right hand plot is a density from the first one, and the thresholds are generated either using [dropletUtils](https://bioconductor.org/packages/release/bioc/html/DropletUtils.html) or by the method described in that discussion. We could use any of these thresholds to select cells, assuming that anything with fewer counts is not a valid cell. By default, Alevin does something similar, and we can learn something about that by plotting just the barcodes Alevin retains. Go back and re-run the droplet barcode plot, this time selecting:
+ * MTX input (quants_mat.mtx.gz).
+ * You will need to select the option to assume cells are by row (more on that later).    
 
-![droplet barcode plot tool](droplet_barcode_tool2.png)
+![droplet barcode plot tool](droplet_barcode_tool2-2.png)
 
-This will use the actual sum of cell-wise counts produced in Alevin's outputs to make the plot. The output will be like:
+This will use the actual cell counts produced in Alevin's outputs to make the plot. The output will be like:
 
 ![barcode plot from processed barcode counts](barcodes_processed.png)
 
-You should see a completely vertical drop-off where Alevin has trunctated the distribution.
+You should see a completely vertical drop-off where Alevin has trunctated the distribution (excluded any cell barcode that had <10 UMI).
```

Comment:
```suggestion
You should see a completely vertical drop-off where Alevin has trunctated the distribution.
```
This will be the threshold Alevin picked off the curve, not 10 UMI.

Reply:
'Truncated' will be meaningless to a wet-labber. However there is a default threshold of 10 UMI first, and then it will pick a threshold based off the curve, right?

Reply:
Happy for you to use a synonym for truncated- but the reason the plot looks as it does is because Alevin has picked a threshold, not due to any parameter we input.

Reply:
OK! But to be clear, the 'default threshold: 10' is not doing anything in this scenario?

Reply:
That's just the bare minimum, in most cases Alevin will be picking a point further up the distribution than that.